packages/next/next-server/lib/utils.ts

```diff
@@ -45,6 +45,18 @@ export type AppTreeType = ComponentType<
   AppInitialProps & { [name: string]: any }
 >
 
+/**
+ * Web vitals provided to _app.reportWebVitals by Core Web Vitals plugin developed by Google Chrome team.
+ * https://nextjs.org/blog/next-9-4#integrated-web-vitals-reporting
+ */
+export type NextWebVitalsMetricType = {
```

Comment:
Feel free to just name it `NextWebVitalsMetric`, re-exporting the type with a different name (in the change added to `pages/_app.tsx`) can be annoying while trying to navigate references in an editor.